src/fetch.js

```diff
@@ -1,6 +1,28 @@
-let fetch = require('isomorphic-unfetch')
+import addVersion from './api/addVersion'
+
+// Here we patch fetch and XMLHttpRequest to always add version parameter to api calls so that cached results
+// from previous versions of the app aren't served to new versions.
+/* istanbul ignore else */
+if (typeof window !== 'undefined') {
+  const originalFetch = window.fetch
+
+  window.fetch = function rsfVersionedFetch(url, init) {
+    return originalFetch(addVersion(url).toString(), init)
```

Comment:
This patch needs to handle the first param being a request object. https://developer.mozilla.org/en-US/docs/Web/API/Request

Reply:
done